conf/test_nanopore.config

```diff
@@ -23,4 +23,6 @@ params {
     input           = 'https://raw.githubusercontent.com/Arcadia-Science/test-datasets/main/metagenomics/ont/samplesheet_test.csv'
     platform        = 'nanopore'
     sourmash_dbs    = 'https://raw.githubusercontent.com/Arcadia-Science/test-datasets/main/metagenomics/sourmash_dbs/sourmash_dbs_test.csv'
+    diamond_db      = 'https://github.com/Arcadia-Science/test-datasets/raw/main/metagenomics/diamond_dbs/test-cheese-proteins-db.dmnd'
+    diamond_columns = 'qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore corrected_bitscore qlen slen'
```

Comment:
```suggestion
    diamond_columns = 'qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore corrected_bitscore qlen slen qcovs qcovhsp'
```

Reply:
I tried this out and decided to leave these out because it threw an error with the test data, so the default options and test profile diamond columns are the simplest set of output columns